Nadim.CinemaReservationSystem.Web/ClientApp/src/components/Cinema/Cinema.js

```diff
@@ -332,13 +362,30 @@ export default class Cinema extends Component{
         }
     }
 
+    renderAlertMessage(){
+        return(
+            <Alert
+                bsStyle={this.state.alertStyle}
+                onDismiss={() => this.setState({show: false})}
+            >
+                <div className="font-bold-x-large">
+                    Something happened
```

Comment:
"Something happened" is bad choice of message title to be used both for errors and notifications. You should use appropriate message title for different situations.